student.rb

```diff
@@ -3,12 +3,12 @@
 class Student < Person
   attr_accessor :classroom
 
-  def initialize(name = "Unknown", age = 0, parent_permission = true, classroom = "Unknown")
-    super(name, age, parent_permission)
+  def initialize(name = 'Unknown', classroom = 'Unknown', parent_permission: true)
+    super(name, parent_permission)
     @classroom = classroom
   end
 
```

Comment:
- [ ] Please pass age in the `initializer` method and in the `super` method as parameter. So while creating students users should be able to provide their age.